doc/includes/API.md

```diff
@@ -6186,6 +6193,46 @@ Type|Description
 
 
 
+#### modifierNotFound
+
+```js
+class BaseModel extends Model {
+  static modifierNotFound(builder, modifier) {
+    const { properties } = this.jsonSchema
+    if (properties && modifier in properties) {
+      builder.select(modifier)
+    } else {
+      super.modifierNotFound(builder, modifier)
+    }
+  }
+}
+```
+
+Handles modifiers that are not recognized by the various mechanisms that can specify
+them, such as [`modify`](#modify) and [`applyModifier`](#applymodifier), as well as
+the use of modifiers in eager expressions (see [`RelationExpression`](#relationexpression))
+and in relations (see [`RelationMapping`](#relationmapping).
+
+By default, the static `modifierNotFound()` hook throws a `ModifierNotFoundError` error.
+If a model class overrides the hook, it can decide to handle the modifer through the passed
+`builder` instance, or call the hook's definition in the super class to still throw the error.
+
+##### Arguments
+
+Argument|Type|Description
+--------|----|-------------------
+builder|[`QueryBuilder`](#querybuilder)|The query builder on which to apply the modifier.
+modifier|string|The name of the unknown modifier.
+
+##### Return value
+
+Type|Description
+----|-----------------------------
+boolean|`true` if the unknown modifier was handled, `false` otherwise, indicating an error should be thrown.
+
```

Comment:
Isn't this now wrong?

Reply:
Yes indeed, I forgot to remove that. When there's no return value at all, I just remove the list, yes?